packages/@ember/-internals/glimmer/lib/helpers/mut.ts

```diff
@@ -1,10 +1,10 @@
 /**
 @module ember
 */
-import { symbol } from '@ember/-internals/utils';
 import { assert } from '@ember/debug';
 import { VMArguments } from '@glimmer/interfaces';
-import { PathReference, UPDATE_REFERENCED_VALUE } from '@glimmer/reference';
+import { createInvokableRef, isUpdatableRef } from '@glimmer/reference';
+import { _WeakSet } from '@glimmer/util';
```

Comment:
Hmm, this seems unused? How is linting happy?

Reply:
Unsure, removed the import though. Could be because it has an underscore in front of it?

Reply:
Ya, possibly due to the underscoring.